_posts/2023-01-18-11.0.0-release.md

```diff
@@ -0,0 +1,119 @@
+* Exposed FlightService_ServiceDesc and RegisterFlightServiceServer to allow easily incorporating a flight service into an existing gRPC server (GH-15174)[https://github.com/apache/arrow/issues/15174]
+
+### Arrow
+* Function `ApproxEquals` was implemented for scalar values (GH-29581)[https://github.com/apache/arrow/issues/29581]
+* `UnmarshalJSON` for the `RecordBuilder` now properly handles extra unknown fields with complex/nested values (GH-31840)[https://github.com/apache/arrow/issues/31840]
+* Decimal128 and Decimal256 type support has been added to the CSV reader (GH-33111)[https://github.com/apache/arrow/issues/33111]
+* Fixed bug in `array.UnionBuilder` where `Len` method always returned 0 (GH-14775)[https://github.com/apache/arrow/issues/14775]
+* Fixed bug for handling slices of Map arrays when marshalling to JSON and for IPC (GH-14780)[https://github.com/apache/arrow/issues/14780]
+* Fixed memory leak when compressing IPC message body buffers (GH-14883)[https://github.com/apache/arrow/issues/14883]
+* Added the ability to easily append scalar values to array builders (GH-15005)[https://github.com/apache/arrow/issues/15005]
+
+#### Compute
+* Scalar binary (add/subtract/multiply/divide/etc.) and unary arithmetic (abs/neg/sqrt/sign/etc.) has been implemented for the compute package  (GH-33086)[https://github.com/apache/arrow/issues/33086] this includes easy functions like `compute.Add` and `compute.Divide` etc.
+* Scalar boolean functions like AND/OR/XOR/etc. have been implemented for compute (GH-33279)[https://github.com/apache/arrow/issues/33279]
+* Scalar comparison function kernels have been implemented for compute (equal/greater/greater_equal/less/less_equal) (GH-33308)[https://github.com/apache/arrow/issues/33308]
+* Scalar compute functions are compatible with dictionary encoded arrays by casting them to their value types (GH-33502)[https://github.com/apache/arrow/issues/33502]
+
+### Parquet
+* Panic when decoding a delta_bit_packed encoded column has been fixed (GH-33483)[https://github.com/apache/arrow/issues/33483]
+* Fixed memory leak from Allocator in `pqarrow.WriteArrowToColumn` (GH-14865)[https://github.com/apache/arrow/issues/14865]
+* Fixed `writer.WriteBatch` to properly handle writing encrypted parquet columns and no longer silently fail, but instead propagate an error (GH-14940)[https://github.com/apache/arrow/issues/14940]
+
+## Java notes
+
+## JavaScript notes
+
+* Bugfixes and dependency updates.
+* Arrow now requires BigInt support. [GH-33681](https://github.com/apache/arrow/pull/33682)
+
+## Python notes
+
+New features and improvements:
+
+* Numpy conversion for ListArray is improved taking into account sliced offset [(GH-20512)](https://github.com/apache/arrow/issues/20512)
+* DataFrame Interchange Protocol is implemented for ``pyarrow.Table`` ([GH-33346](https://github.com/apache/arrow/issues/33346)).
+
+## R notes
+
+For more on what’s in the 11.0.0 R package, see the [R changelog][4].
```

Comment:
```suggestion
* map_batches() is lazy by default; it now returns a RecordBatchReader instead of a list of RecordBatch objects unless lazy = FALSE. [GH-14521](https://github.com/apache/arrow/issues/14521)
* A substantial reorganisation, rewrite of and addition to, many of the vignettes and README. [GH-14514](https://github.com/apache/arrow/issues/14514)

For more on what’s in the 11.0.0 R package, see the [R changelog][4].
```